awscli/examples/s3/cp.rst

```diff
@@ -90,34 +99,6 @@ Output::
 
     copy: s3://mybucket/test.txt to s3://mybucket/test2.txt
 
-Note that if you're using the ``--acl`` option, ensure that any associated IAM
```

Comment:
I think we want to keep this.  There has been some customer confusion about the permissions needed when using `--acl`.